docs/modules/user-guide/partials/proc_devfile-properties.adoc

```diff
@@ -0,0 +1,258 @@
+[id="proc_devfile-properties_{context}"]
+= Devfile properties 
+
+[role="_abstract"]
+Use this page to work with yaml files of devfile properties. See the following table of contents to go to the devfile property you want to work with:
+
+== SchemaVersion
+
+[source,yaml]
+----
+v2.0
+---
+schemaVersion: 2.0.0
+----
+
+[cols="1,1"]
+|===
+|Key
+|schemaVersion
+
+|Type
+|string
+
+|Required
+|yes
+
+|Description
+|v2.0.0 of the schema
+|===
+
+For more information, go to: xref:migrating-schema-version.adoc[].
```

Comment:
For links like this, it's always good to mention what the information provides more of. "For more information on xy, go to ..."
Also I've been wondering how the xref is really rendered in the end. Here it looks like it displays "migrating-schema-version" while the link should display the name of the section nicely, right? Like the headline of the adoc file e.g. "Migrating schema versions", you know what I mean?
 I am not sure about your workflow though, and I'm sure you've done what you've seen elsewhere.

This comment relates to the other additional resources as well, but I will just comment it here now :) 


This goes for the other additional resources as well, but I will just comment it here now :)

Reply:
In my docs it looks like this xref:xyz[name]
e.g. xref:developing-for-the-cloud-with-openshift-in-devstudio_getting-started-with-container-and-cloud-based-development[Developing for the Cloud with OpenShift in {devstudio}].
And then what's in [] will be displayed.

Reply:
Thanks, Eva! I'll add more context to the links, then have you review. When I build my docs locally, I'll send you a screenshot of what it looks like. On my local hosts, it renders nicely, but I'll see what you think too! And yes, it's frustrating that it won't render properly in it's raw form, haha

Reply:
Screenshot of how the xrefs look when rendered:
<img width="999" alt="Screen Shot 2021-04-30 at 11 14 23 AM" src="https://user-images.githubusercontent.com/70717303/116716046-48bd5e00-a9a5-11eb-895d-27656b5546fe.png">